server/db/foods.js

```diff
@@ -22,5 +38,6 @@ function getFood (id, db = connection) {
 
 module.exports = {
   getFoods,
+  addFood,
```

Comment:
Swap order of this with the below fn. Thanks!

Reply:
done